bokeh/core/enums.py

```diff
@@ -158,9 +158,6 @@ def enumeration(*values, **kwargs):
 #: Specify an aggregation type for different charts
 Aggregation = enumeration("sum", "mean", "count", "nunique", "median", "min", "max")
 
-#: Specify the language used in a CustomJS callback
-ScriptingLanguage = enumeration("javascript", "coffeescript")
-
```

Comment:
My policy is to get rid of old code in the first place, to make sure we use new APIs everywhere and then deprecate. I will do it later.